### PR TITLE
[Bugfix:InstructorUI] Fixed Improper use of Date strings in JS

### PR DESCRIPTION
**Pull request by mahinigam**
### Please check if the PR fulfills these requirements:

* [x] Tests for the changes have been added/updated (if possible)
* [x] Documentation has been updated/added if relevant
* [x] Screenshots are attached to Github PR if visual/UI changes were made

#### Currently in our JavaScript code, there are numerous calls to the Date constructor which pass in a date string.
Many of the strings passed in do not follow the ISO8601 format and rely on non-standard browser behavior to parse the date.
This is very bad practice and can lead to parts of the site breaking across different browsers.

To fix this, all date strings in JavaScript should be updated to be formatted properly.

Ex.) new Date("02/01/1970 05:00") => new Date("1970-02-01T05:00:00")

### CODE REVIEW

File: site/public/js/latedays.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ $(document).ready(() => {
                             date = new Date();
                             break;
                         case 1:
-                            date = new Date('9998-01-01');
+                            date = new Date("9998-01-01");
```
Comment (williamjallen) [inline site/public/js/latedays.js:40]: This change isn't necessary and should be reverted.

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: site/public/js/latedays.js (hunk 1 above)
Comment (mahinigam-28) [reply site/public/js/latedays.js:40]: done

**Review by ziesski — COMMENTED** [review]
Please resolve the javascript lint error (double quotation->single quotation)

**Review by VVVITH — APPROVED** [review]
After I fix the branch conflict, the code looks good now. So I approve it, but I'm not sure is this change a good change and make real improve? More reviews needed.

**Review by cjreed121 — APPROVED** [review]

**Pull request merged**